.travis.yml

```diff
@@ -115,6 +115,24 @@ matrix:
         - python tools/make.py -t GCC_ARM -m DISCO_F401VC --source=. --build=BUILD/DISCO_F401VC/GCC_ARM -j0
         # Run local equeue tests
         - make -C $EVENTS/equeue test
+        # Run profiling tests
+        - make -C $EVENTS/equeue prof | tee prof
+      after_success:
+        # update status if we succeeded, compare with master if possible
+        - |
+          CURR=$(grep -o '[0-9]\+ cycles' prof | awk '{sum += $1} END {print sum}')
+          PREV=$(curl https://api.github.com/repos/$TRAVIS_REPO_SLUG/status/master \
+              | jq -re "select(.sha != \"$TRAVIS_COMMIT\")
```

Comment:
jq: Command-line JSON parser.

Good to know.

Reply:
It's not on all Ubuntu distros yet, but Travis guarantees it's their machines.

Reply:
As long as Travis guarantees it :+1: 

Also, I take it that's one of the reasons why `|| echo 0` exists

Reply:
oh, that's a trick on my end. Since we're calculating a ratio below, if we fetch an old value of 0, we would divide by zero.

So for any errors, we just treat it as zero, and check it later.